data_pump/utils.py

```diff
@@ -7,14 +7,15 @@
 from migration_const import MAPPING_PATH
 from const import API_URL
 
+_this_dir = os.path.dirname(os.path.abspath(__file__))
 
 def read_json(file_name, file_path=DATA_PATH):
     """
     Read data from file as json.
     @param file_name: file name
     @return: data as json
     """
-    f_path = os.path.join(file_path, file_name)
+    f_path = os.path.join(_this_dir, '..', file_path, file_name)
```

Comment:
Are you sure about this? Is it done like this in other places?

Reply:
when we read json, we use only this method... and this advised me Pavol, it's way how to avoid mishmash between folder from which we run script and where we can find files

Reply:
alright :)